geonode_mapstore_client/client/js/plugins/MapViewersCatalog.jsx

```diff
@@ -123,7 +124,7 @@ function MapViewersCatalogPlugin({
                         </div>
                         <div className="gn-new-map-viewer-action-card">
                             <div >
-                                <Icon glyph="cogs" className="fa-4x"/>
+                                <Glyphicon glyph="context" className="fa-4x"/>
```

Comment:
Do we need the fa-4x class? Could we try to use the Text component with font size xl as wrapper instead?